crawl-ref/source/dat/database/miscname.txt

```diff
@@ -136,6 +136,14 @@ The Orb of Zot is coveted for many reasons. Will you seize it for your own?
 A smirking vizier told you that the Orb of Zot is just within this dungeon.
 
 If you return without finding the Orb of Zot, so far below, it's all over!
+
+One, two, three... Ready or not, find the Orb of Zot!
```

Comment:
I get what you're going for here, but this is a bit out of tone for the game. Crawl can be silly, but it's not usually playground-rhymes silly. Every welcome message could be the very first thing a new player sees, and this isn't a great introduction for them.